src/components/sidebar/components/sidebar-filter-input/style.module.css

```diff
@@ -47,7 +47,7 @@
 .filterInput {
   composes: focus-border-and-box-shadow-light from global;
   background-color: transparent;
-  border: 1px solid white;
+  border-color: var(--token-elevation-inset-border-color);
```

Comment:
Adds border to filter input, to match mockups.

Reply:
So two things here:

- The border of the input is being handled by the `box-shadow` below, and that came from the Figma designs
- the 1px white border is needed in the input's default state so that it doesn't shift when receiving focus

Reply:
Ah nice, seeing this now, reverted this 👍 thank you! 💖